support/utils/kubernetes.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/e2e-framework/klient/conf"
+)
+
+func FetchKubernetesClients(config string) (cfg *rest.Config, kubernetesClient kubernetes.Interface, dynamicClient dynamic.Interface, err error) {
```

Comment:
I think we should think about simplifying this, may be break it into two separate functions to return the K8s client needed (i.e. `FetchDynamicClient` and `FetchTypedClient` or something similar).

Reply:
I will get rid of this entirely in favor of keeping the interface simple and have the interface return the `rest.Config` that one can use to create whatever client they need.